### ISSUE TITLE
getTemplates() throws a fatal error

### ISSUE THREAD

**Issue by michelaerts** [flowmailer/flowmailer-php-sdk#3]
`$flowmailer->getTemplates();` throws an error when using it like in this example: https://flowmailer.net/apidoc/flowmailer-api#get_account_id_templates.

`Fatal error: Uncaught Exception: The type of the "data" attribute for class "Flowmailer\API\Model\Template" must be one of "string" ("null" given).`

The dump (line 2108 in `Endpoints.php`) of the `$response` just before it gets deserialized looks like this:
```
object(Flowmailer\API\Serializer\ResponseData)#12998 (2) {
  ["responseBody":"Flowmailer\API\Serializer\ResponseData":private]=>
  string(374) "[
    {"id":"1","description":"Event to JSON","mimeType":"flowmailer/json","templateEngine":"freemarker-2.3.20","data":null},
    {"id":"2","description":"Template1","mimeType":"text/html","templateEngine":"freemarker-2.3.20","data":null},
    {"id":"3","description":"Template2","mimeType":"text/html","templateEngine":"freemarker-2.3.20","data":null}
  ]"
  ["meta":"Flowmailer\API\Serializer\ResponseData":private]=>
  array(0) {
  }
}
```
As you can see the `data` attribute is `null`.

When using `$flowmailer->getTemplate(1);` the `data` attribute is not `null` but contains the content like in the dashboard.

```
object(Flowmailer\API\Model\Template)#13032 (10) {
  ["contentId":"Flowmailer\API\Model\Template":private]=>
  NULL
  ["data":"Flowmailer\API\Model\Template":private]=>
  string(104) "<#if event??>
    <#assign archive = fetchMessageArchive(event.message.id) />

${toJson(.main)}"
  ["decodeBase64":"Flowmailer\API\Model\Template":private]=>
  NULL
  ["description":"Flowmailer\API\Model\Template":private]=>
  string(13) "Event to JSON"
  ["disposition":"Flowmailer\API\Model\Template":private]=>
  NULL
  ["doNotUpdateOnCopy":"Flowmailer\API\Model\Template":private]=>
  NULL
  ["filename":"Flowmailer\API\Model\Template":private]=>
  NULL
  ["id":"Flowmailer\API\Model\Template":private]=>
  string(6) "1"
  ["mimeType":"Flowmailer\API\Model\Template":private]=>
  string(15) "flowmailer/json"
  ["templateEngine":"Flowmailer\API\Model\Template":private]=>
  enum(Flowmailer\API\Enum\TemplateEngine::FREEMARKER_2_3_20)
}
```
Tnx.

**Comment by reyostallenberg** [comment]
Hi @michelaerts ,

I've fixed this issue for you in version 2.0.1, see https://github.com/flowmailer/flowmailer-php-sdk/commit/dfa10ba2a961c1f755954734ada27dac5682e809 
Thanks for your issue, please contact us again if you have another problem.

Kind regards,
Reyo Sallenberg
Spotler Flowmailer

**Comment by michelaerts** [comment]
Hi @reyostallenberg 
tnx, it works!
